### PR TITLE
fix: small updates to cheqd module and demo

### PR DESCRIPTION
**Pull request by TimoGlastra**

### CODE REVIEW

**Comment by TimoGlastra** [comment]
Draft as there's one failing test with the cheqd moduel

File: package.json [hunk 1]
```diff
@@ -60,9 +60,10 @@
     "ws": "^8.13.0"
   },
   "resolutions": {
-    "@types/node": "^16.11.7"
+    "@types/node": "^16.11.7",
+    "ref-napi": "npm:@2060.io/ref-napi"
```
Comment (genaris) [inline package.json:64]: I guess the ref-napi trick must still be left to the CI script, right?

File: yarn.lock [hunk 2]
```diff
@@ -10356,10 +10356,10 @@ ref-array-di@1.2.2, ref-array-di@^1.2.2:
     array-index "^1.0.0"
     debug "^3.1.0"
 
-ref-napi@3.0.3, "ref-napi@^2.0.1 || ^3.0.2", ref-napi@^3.0.3:
-  version "3.0.3"
-  resolved "https://registry.yarnpkg.com/ref-napi/-/ref-napi-3.0.3.tgz#e259bfc2bbafb3e169e8cd9ba49037dd00396b22"
-  integrity sha512-LiMq/XDGcgodTYOMppikEtJelWsKQERbLQsYm0IOOnzhwE9xYZC7x8txNnFC9wJNOkPferQI4vD4ZkC0mDyrOA==
+ref-napi@3.0.3, "ref-napi@^2.0.1 || ^3.0.2", ref-napi@^3.0.3, "ref-napi@npm:@2060.io/ref-napi":
```
Comment (genaris) [inline yarn.lock:10359]: Here as well, resolve to regular ref-napi unless working on node 18.

File: DEVREADME.md [hunk 3]
```diff
@@ -74,6 +74,14 @@ docker exec indy-pool add-did-from-seed 000000000000000000000000Trustee9 TRUSTEE
 # docker exec indy-pool add-did "NkGXDEPgpFGjQKMYmz6SyF" "CrSA1WbYYWLJoHm16Xw1VEeWxFvXtWjtsfEzMsjB5vDT"
 ```
 
+### Setup Cheqd Ledger
+
+In addition, there's also a docker command to run a cheqd test network.
```
Comment (genaris) [inline DEVREADME.md:79]: Would be good to also have an advice about skipping cheqd tests like we did for postgres

**Review by genaris — APPROVED** [review]

**Pull request merged**